ballerina/tests/basic-tests.bal

```diff
@@ -279,18 +275,16 @@ function testComplexTypesWithExecute() returns error? {
         quantity: 5
     };
     MedicalNeedClient mnClient = check new ();
-    int? id = check mnClient->create(need);
-    test:assertTrue(id is int);
+    MedicalNeed need2 = check mnClient->create(need);
+    test:assertTrue(need2.needId is int);
 
-    if id is int {
-        int count = 0;
-        _ = check from MedicalNeed need2 in mnClient->execute(` WHERE itemId = 1`)
-            do {
-                test:assertEquals(need2.itemId, 1);
-                count = count + 1;
-            };
-        test:assertTrue(count > 0);
-    }
+    int count = 0;
+    _ = check from MedicalNeed need3 in mnClient->execute(` WHERE itemId = 1`)
```

Comment:
Are we giving a compiler error, if there is any usage of `execute` function in user code

Reply:
@niveathika

Reply:
Actually we cant. The code modifying happens before the compiler validations. At that point we cannot differentiate between the two use cases. I was thinking of adding a warning in the API doc of generated clients